AsyncDisplayKit/ASButtonNode.mm

```diff
@@ -195,8 +195,8 @@ - (ASLayoutSpec *)layoutSpecThatFits:(ASSizeRange)constrainedSize
   ASStackLayoutSpec *stack = [[ASStackLayoutSpec alloc] init];
   stack.direction = self.laysOutHorizontally ? ASStackLayoutDirectionHorizontal : ASStackLayoutDirectionVertical;
   stack.spacing = self.contentSpacing;
-  stack.justifyContent = ASStackLayoutJustifyContentCenter;
-  stack.alignItems = ASStackLayoutAlignItemsCenter;
+  stack.horizontalAlignment = self.contentHorizontalAlignment ? self.contentHorizontalAlignment : ASAlignmentMiddle;
```

Comment:
@1nput0utput can you set _contentHorizontalAlignment, and _contentVerticalAlignment, to the default values in the -init method for the button?  (use the ivars, not the property for efficiency).  Then here, set the stack properties directly to the ivar values without checking them.  Nothing guarantees that the enum values won't be zero anyway, so I'm not sure it is ideal to use the ternary operator to check if they have been set to non-default values.